internal/cmd/egctl/translate.go

```diff
@@ -363,6 +368,7 @@ func TranslateGatewayAPIToXds(namespace, dnsDomain, resourceType string, resourc
 			GlobalRateLimit: &translator.GlobalRateLimitSettings{
 				ServiceURL: ratelimit.GetServiceURL(namespace, dnsDomain),
 			},
+			Logger: logging.DefaultLogger(os.Stdout, egv1a1.LogLevelInfo),
```

Comment:
We need to initialize a logger for  the xds translator, otherwise the gectl may crush when translation fails.